prow/proxy-common.inc

```diff
@@ -31,7 +31,8 @@ GOPATH=/home/prow/go
 ROOT=/go/src
 
 # Configure available resources and disable IPv6 tests.
-export BAZEL_BUILD_ARGS="--verbose_failures --test_env=ENVOY_IP_TEST_VERSIONS=v4only --test_output=errors"
+BAZEL_BUILD_ARGS="${BAZEL_BUILD_EXTRA_ARGS:-} --verbose_failures --test_env=ENVOY_IP_TEST_VERSIONS=v4only --test_output=errors"
```

Comment:
Why not just append to BAZEL_BUILD_ARGS?

Reply:
Are you asking me to use "+=" instead of interpolation? Bazel is finicky about whitespace between arguments and I'll still need to trim the string to keep it happy (using [`${...# } or ${...% }`](https://www.gnu.org/software/bash/manual/html_node/Shell-Parameter-Expansion.html#:~:text=the%20last%20element.-,%24%7Bparameter%23word%7D,-%24%7Bparameter%23%23)).

Reply:
If you're saying, assume `BAZEL_BUILD_ARGS` might have something interesting already in it, and expose that as the injection point: I think saying `EXTRA` in the variable name helps the user understand this isn't all that will be passed to bazel, it's what goes in above and beyond the defaults.